addons/sale_timesheet/views/account_invoice_views.xml

```diff
@@ -1,21 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
 
-    <record id="action_timesheet_from_invoice" model="ir.actions.act_window">
-        <field name="name">Timesheet</field>
-        <field name="type">ir.actions.act_window</field>
-        <field name="res_model">account.analytic.line</field>
-        <field name="view_mode">tree,form,graph</field>
-        <field name="context">{'search_default_timesheet_invoice_id': active_id}</field>
```

Comment:
what about just change the domain of the xml action ? isn't it working ?

Reply:
working, changed